tests/schema/tests.py

```diff
@@ -2403,6 +2403,20 @@ class Meta:
         student = Student.objects.create(name='Some man')
         doc.students.add(student)
 
+    @unittest.skipUnless(connection.vendor == 'oracle', 'Oracle specific sequence name truncation')
+    def test_oracle_sequence_name_truncation_regression(self):
+        """
+        Regression test for #28451.
+        Oracle-specific.
+        """
+        with connection.schema_editor() as editor:
+            editor.create_model(AuthorWithEvenLongerName)
+
+        # Make sure the sequence name matches what we expect based on default truncation rules
```

Comment:
Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style), e.g. “Loop over” rather than “We loop over”.

If the docstring stated the expected behavior, I don't think an inline comment is needed.